script.js

```diff
@@ -126,34 +192,36 @@ function makePageForEpisodes(episodeList) {
 
     containerElement.appendChild(episodeContainer);
   });
+  const searchContainer = getElementById("search-container");
+  searchContainer.innerHTML = "";
+  searchContainer = searchFunction(episodeList);
 }
 
-
-
 function makeFooter(allEpisodes) {
   const footer = document.createElement("footer");
   footer.className = "footer-container";
-  footer.innerHTML = 
-    `Data Collected from TVMaze.com
+  footer.innerHTML = `Data Collected from TVMaze.com
     <br>
     Found ${allEpisodes.length} episode(s)`;
   document.body.appendChild(footer);
 }
 
+async function initialise() {
+  const allShows = await fetchAllShows();
+  const allEpisodes = await fetchAllEpisodes(1);
 
-async function initialise () {
-  // const allShows = await fetchAllShows()
-  // console.log(allShows);
+  makeHeader(allEpisodes);
 
-  const allEpisodes = await fetchAllEpisodes(82);
-  console.log(allEpisodes);
+  populateShowSelect(allShows);
+  populateEpisodesSelect(allEpisodes);
 
-  makeHeader(allEpisodes);
-  makePageForEpisodes(allEpisodes);
+  searchFunction(allEpisodes);
```

Comment:
Search functionality seems to be broken on the deployed Netlify version. It shows '[object HTMLDivElement]' instead of the search box.